scripts/cloud-api-demo/correct_permissions.py

```diff
@@ -0,0 +1,49 @@
+import argparse
```

Comment:
This must be the first line to specify the specific interpreter to use

```
#!/usr/bin/env python3
```

Reply:
Added